frontend/www/js/omegaup/components/problem/CollectionDetails.vue

```diff
@@ -1,20 +1,63 @@
 <template>
   <div>
     <h1 class="card-title">{{ title }}</h1>
+    <div class="row">
+      <div class="col col-md-4">
+        <div>
+          <omegaup-problem-filter-tags
+            :tags="collectionTags"
+            :public-tags="publicTags"
+            :selected-tags="selectedTags"
+            @add-tag="addTag"
+            @check="(selectedTags) => check(selectedTags)"
```

Comment:
De acuerdo a los comentarios de Luis Héctor, posiblemente sea mejor dejar de manejar los tags seleccionados y todo lo que tenga que ver con agregar tags, y mejor se haga todo desde el otro componente. Así que aquí debería quedar algo así:
```suggestion
            :tags.sync="tags"
            :public-tags="publicTags"
```